content/blog/2020-06-23-finding-errors-in-data-analyses/index.Rmarkdown

```diff
@@ -0,0 +1,68 @@
+---
+title: "Finding Errors in Data Analyses"
+author: 
+  - "[Ian Flores](https://education.rstudio.com/trainers/people/flores_siaca+ian/)"
+date: '2020-06-23'
+tags:
+  - internship
+slug: finding-errors-in-data-analyses
+description: |
+  A preview of a research project studying how and how well data scientists find errors.
+photo:
+  author: Ian Flores
+---
+
+While some people guide their teaching practices with anecdotal evidence,
+we prefer to study these practices through a scientific lens.
+The goal of my summer internship with the RStudio Education team is
+to find out how and how well data scientists and data science students identify errors in data analyses.
+In particular,
+we want to explore how the ability to catch errors varies by level of experience
+as well as familiarity to the data and type of analysis. 
+
+To do this we have developed RMarkdown documents
```

Comment:
```suggestion
To do this we have developed R Markdown documents
```
I tend to use rmarkdown when referring to the package, then R Markdown for the kind of document